codegen/src/main/twirl/templates/JavaServer/Handler.scala.txt

```diff
@@ -92,6 +92,27 @@ public class @{serviceName}HandlerFactory {
       return partial(implementation, prefix, SystemMaterializer.get(system).materializer(), eHandler, system);
     }
 
+@if(serviceName == "ServerReflection") {} else {
```

Comment:
Don't add the builder if the service we're processing is `ServerReflection`. #turtles

Reply:
Does it hurt to have it available for reflection itself as well?

Reply:
It doesn't. It would just be confusing. 😅